Add 1.13 release notes (#497)
okteto/docs: src/content/self-hosted/install/releases.mdx

id: releases
---

## 1.13.0

27 September 2023
This version is compatible with Kubernetes versions 1.24 to 1.26.

### Breaking Changes

> A valid license is now required for okteto to work. If you were running a previous version of okteto that has no license, an expired license or no license at all make sure to [reach out to us](https://www.okteto.com/contact/) to request one, [open a support ticket](https://okteto.com/support?_gl=1*1ikydzg*_ga*MTMyMjc0MTYyOS4xNjUyMjA1MjQ0*_ga_KSKZWJHTJZ*MTY5NTM5Njk3MS4yMTguMS4xNjk1Mzk3MDk3LjU2LjAuMA..) or signup for our [30 day free trial](https://www.okteto.com/free-trial/).

Reviewer: We already reached out to all the customers, so the support ticket is not need

```suggestion
Okteto requires you to use a valid license as part of the installation. If you were running a previous version of Okteto and you don't have a valid license, signup for our [30 day free trial](https://www.okteto.com/free-trial/)  or [reach out to us](https://www.okteto.com/contact/).
```

> A valid license is now required for okteto to work. If you were running a previous version of okteto that has no license, an expired license or no license at all make sure to [reach out to us](https://www.okteto.com/contact/) to request one, [open a support ticket](https://okteto.com/support?_gl=1*1ikydzg*_ga*MTMyMjc0MTYyOS4xNjUyMjA1MjQ0*_ga_KSKZWJHTJZ*MTY5NTM5Njk3MS4yMTguMS4xNjk1Mzk3MDk3LjU2LjAuMA..) or signup for our [30 day free trial](https://www.okteto.com/free-trial/).

Reviewer: I think we should also mention that when the trial license expire, the instance is locked.

Reviewer 2: Agree, that is also a breaking change and it should be included

### Improvements

- Updates to [**Cloud Secret**](https://www.okteto.com/docs/self-hosted/administration/configuration/#cloud-provider-secret) now take immediate effect  <!-- 7032 -->

> - The Okteto CLI used for Pipeline installation has been upgraded to version [2.20.0](https://github.com/okteto/okteto/releases/tag/2.20.0)

Reviewer 2: @omnomagonz @rberrelleza Should we mention something here about kubetoken and dynamic tokens? This is the first chart release which includes the changes

Reviewer: Yes I think it's something thst benefits the user

> - Allow to configure kubetoken auth TTL through helm configuration (defaults to 24h)  <!-- 7142 -->

Reviewer 2: Could you add a link to this section? https://www.okteto.com/docs/1.13/self-hosted/administration/configuration/#kubetoken
